src/actors/tornado_scanner/tornado_scanner.py

```diff
@@ -0,0 +1,91 @@
+import json
+import os
+import re
+import sys
+import ast
+
+
+def is_tornado_package(main_file_path):
+    with open(main_file_path, 'r') as f:
+        tree = ast.parse(f.read())
+        for node in ast.walk(tree):
+            if isinstance(node, (ast.Import, ast.ImportFrom)):
+                for elem in node.names:
+                    if 'tornado' in elem.name:
+                        return True
+    return False
+
+
+def get_python_exec(proc_cmdline, virtual_env_path=None):
+    if virtual_env_path is not None:
+        v_python_exec = os.path.join(app_metadata['virtual_env_path'], 'bin/python')
+        return os.readlink(v_python_exec) if os.path.islink(v_python_exec) else 'python'
+
+    python_exec = re.search("python([0-9]\.?[0-9]?(d|m|u)?)?", process['cmdline'])
+    return python_exec.group() if python_exec is not None else None
+
+
+def get_main_file_path(cmdline, cwd):
+    main_file = re.search("[a-zA-Z0-9\/]*\.py", cmdline)
+    return os.path.join(cwd, main_file.group()) if main_file is not None else None
+
+
+def extract_processes_param(params):
+    try:
+        return params['process_list'][0]['processes']
+    except LookupError:
+        sys.stderr.write("Could not find processes to check")
+        return None
+
+
+def check_if_files_exist(path, files):
+    result = {}
+    for f in files:
+        result[f] = os.path.exists(os.path.join(path, f))
```

Comment:
In this case I think it's better to use `isfile` instead of `exists`, since the latter returns `True` if the parameter is a directory as well.

Reply:
fixed